x-pack/platform/plugins/shared/alerting/server/lib/rule_gaps/update/calculate_gaps_state.ts

```diff
@@ -40,9 +41,10 @@ export const calculateGapStateFromAllBackfills = async ({
     if ('error' in backfill) {
       continue;
     }
+    const scheduledItems = backfill?.schedule.map(toScheduledItem) ?? [];
```

Comment:
do you need to catch any errors from `toScheduledItem` here?

Reply:
Good catch! I have just added exception handling logic for this 🙏